File: ElectronicNotebook/ElectronicNotebook/ShowNotesCommand.cs
using System;

namespace ElectronicNotebook
{
    /// <summary>
    /// Class-command shows all the notes.
    /// </summary>
    public class ShowNotesCommand: ICommand
    {
        NoteBookReceiver receiver;
        public ShowNotesCommand(NoteBookReceiver rec)
        {
            receiver = rec;
        }
        /// <summary>
        /// Overrided Execute method for ShowNotesCommand.
        /// </summary>
        public void Execute()
        {
            new NoteBookConsoleView().Print(receiver.ShowNoteBookNotes());
        }
    }
}

File: NoteBook/src/main/java/ShowNotesCommand.java
/**
 * This class realize method, that show notes from a Command interface.
 */
public class ShowNotesCommand implements Command {
  NoteBook noteBook;
  public ShowNotesCommand(NoteBook noteBook) {
    this.noteBook = noteBook;
  }

  /**
   * This method is override execute method for show notes.
   */
  public void execute() {
    System.out.println(noteBook.toString());
  }
}